modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-install-kernelrt-rhel-edge_{context}"]
+= Installing the {op-system-rt-kernel} in a {op-system-ostree-first} image
+
+You can include the real-time kernel in a {op-system-ostree} image deployment using image builder.
+
+.Prerequisites
+* You have a Red Hat subscription enabled on the host that includes {op-system-rt-kernel}.
+* You are using the x86_64 architecture.
+* You configured `osbuild` to use the `kernel-rt` repository.
+//TODO insert link after Eliane's MR is published. Should be right around here: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-command-line-interface_composing-a-customized-rhel-system-image#appending-a-kernel_creating-system-images-with-composer-command-line-interface
+
+[NOTE]
+====
+The following example blueprint sections include references gathered from the previous steps required to configure low latency for a {microshift-short} cluster.
+====
+
+[IMPORTANT]
+====
+A subscription that includes the {op-system-rtk} must be enabled on the host used to build the commit.
+====
+
+.Procedure
+
+. Add the following example blueprint sections to your complete installation blueprint for installing the real-time kernel in a {op-system-ostree} image:
++
+[source,text]
+----
+name = "microshift-low-latency"
+description = "RHEL 9.4 and MicroShift configured for low latency"
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-94"
```

Comment:
If we're talking about low-latency specific fragment, we might want to skip this header.

Reply:
WDYT @pmtk Just list packages and skip the header?

Reply:
I think that's alright - we have complete example at the end